### ISSUE TITLE
[Open for all] UI Design for Home page of the website

### ISSUE THREAD

**Issue by Kajol-Kumari** [HITK-TECH-Community/Community-Website#2]
- Make UI for the home page of the website using any of the UI designing tool like figma

**Comment by headphone-rachit** [comment]
I want to do this using HTML,CSS and JS is that ok

**Comment by Kajol-Kumari** [comment]
Hey @headphone-rachit  It's more of like a designing(UI) issue. You don't need to code anything instead you need to make the prototype of home page. We can't directly jump to code until we have a design so we have opened UI issues first to finalize our designs.

**Comment by headphone-rachit** [comment]
ok so first I will create a design which we can follow while coding

**Comment by Kajol-Kumari** [comment]
Yes @headphone-rachit that's right! And you can just add the screenshot of your design in `assets` folder and make the PR for this issue.

**Comment by headphone-rachit** [comment]
ok

**Comment by headphone-rachit** [comment]
![Home Page](https://user-images.githubusercontent.com/65488082/101385687-25a0ed80-38e2-11eb-90f3-7ded12b10246.JPG)

**Comment by headphone-rachit** [comment]
Is this page fine to go ahead

**Comment by headphone-rachit** [comment]
I am not able to upload it in the asset as it is demanding access

**Comment by Kajol-Kumari** [comment]
> I am not able to upload it in the asset as it is demanding access

Hey @headphone-rachit You first need to fork this repo, clone it on your local, make a branch and then add the file using any editor and then make the PR.

**Comment by headphone-rachit** [comment]
I have created the PR, so if it is fine please give me this project.

**Comment by Kajol-Kumari** [comment]
Hey @headphone-rachit Thanks a lot for the PR. But can tou please change the design to something like this. This is a rough overview of what we are expecting in the design. Your creativity is much appreciated😊
![IMG_20201208_160245](https://user-images.githubusercontent.com/44888949/101472989-4b280880-396f-11eb-9232-a373bed9a948.jpg)

**Comment by headphone-rachit** [comment]
I have made some changes and made a PR please check it

**Comment by headphone-rachit** [comment]
if I can start building this page??

**Comment by Kajol-Kumari** [comment]
> if I can start building this page??

Hey @headphone-rachit we need few more designs to finalise it before jumping to code.

**Comment by headphone-rachit** [comment]
ok

**Comment by headphone-rachit** [comment]
i have added one more

**Comment by Aniket31-coder** [comment]
I would like to contribute to this issue!

**Comment by SuNiL-Chau** [comment]
Hi @Kajol-Kumari I would like to design the home page is this issue still on?

**Comment by yashsehgal** [comment]
Hey, I am interested in this issue. Can you assign me this?

**Comment by Kajol-Kumari** [comment]
Hey @Aniket31-coder @SuNiL-Chau @yashsehgal go ahead with this issue. Looking forward to see your awesome designs :)

**Comment by Muskan10Mehta** [comment]
Hey! I am interested in this issue

**Comment by Kajol-Kumari** [comment]
@Muskan10Mehta go ahead with this issue. Looking forward to your design :)

**Comment by abhinavmaharana** [comment]
Can i work on it ? I am interested in this issue

**Comment by Kajol-Kumari** [comment]
sure @abhinavmaharana go ahead with this issue. Looking forward to your design :)

**Comment by abhinavmaharana** [comment]
Thank you @Kajol-Kumari  for assigning me. I want to know few things before starting desiging the ui.
The list is as follows:-
1) Basic rough idea about what are the things you want in the website like any video play some , actually full rough idea of the webiste.
2) Is their any color sceme do you have in mind so that i will try to focus on that on the webiste.
3) What is the purpose of this website (This will help me to choose assets particular to the theme).

This will be enough for if you want to design the landing page only.

**Comment by abhinav-huria** [comment]
I would like to work on this.

**Comment by himanshu007-creator** [comment]
i am working on it!

**Comment by Kajol-Kumari** [comment]
@abhinav-huria you alread have 2 issues on your end so I can't assign this to you @himanshu007-creator go ahead with this issue.

**Comment by himanshu007-creator** [comment]
thanks @Kajol-Kumari for assigning me, can you give me the issue. Whats the deadline till whichj i should design the UI?

**Comment by himanshu007-creator** [comment]
@Kajol-Kumari  mam i have senat a pr . I have added the logo + the home page ui design. Please merge 

https://github.com/HITK-TECH-Community/Community-Website/pull/107

**Comment by himanshu007-creator** [comment]
@Kajol-Kumari  PR for this issue sent https://github.com/HITK-TECH-Community/Community-Website/pull/114

**Comment by SuNiL-Chau** [comment]
Hey @Kajol-Kumari is issue still on? if yes i would like to design this page too

**Comment by Kajol-Kumari** [comment]
> Thank you @Kajol-Kumari for assigning me. I want to know few things before starting desiging the ui.
> The list is as follows:-
> 
> 1. Basic rough idea about what are the things you want in the website like any video play some , actually full rough idea of the webiste.
> 2. Is their any color sceme do you have in mind so that i will try to focus on that on the webiste.
> 3. What is the purpose of this website (This will help me to choose assets particular to the theme).
> 
> This will be enough for if you want to design the landing page only.
> For reference you can check this link https://www.figma.com/file/l5YXbNJuDFMtr3hL5E0nMD/Community-website?node-id=0%3A1

Hey @abhinavmaharana! Sorry i had missed this comment. 

1. Out main motive of making the website is to display the broadcast which may contain few posters and links. 
2. As of now, we have decided to go ahead with [this](https://colors.lol/blocky) pallet
3. Our motive is to reduce the knowledge gap between students in terms of resources and opportunities

**Comment by ritamganguli** [comment]
see sine we are targeting mainly the students of tier-2 and tier-3 can we add a job portal of latest of campus job

**Comment by Kajol-Kumari** [comment]
Yes @ritamganguli that's a good point, we would think and would love to integrate this as well :)

**Comment by kanishkaa24** [comment]
@Kajol-Kumari 
I would like to work on this.

**Comment by Dhriti-Roy** [comment]
I am would like to work on this

**Comment by Kajol-Kumari** [comment]
@Dhriti-Roy we won't be assigning this issue to anyone so `no more assignment` label is added in here. Please look at  the other issues

**Comment by yashsehgal** [comment]
Hey There, This is my approach for webpage design. This is just an experimental UI following a retro-classic theme. Hope you'll like it.

![page-design](https://user-images.githubusercontent.com/62352288/103729904-743bc780-5007-11eb-8531-430acb3696e1.png)

**Comment by Kajol-Kumari** [comment]
> Hey There, This is my approach for webpage design. This is just an experimental UI following a retro-classic theme. Hope you'll like it.
> 
> ![page-design](https://user-images.githubusercontent.com/62352288/103729904-743bc780-5007-11eb-8531-430acb3696e1.png)

Hey @yashsehgal this deisgn is nice and clean! But I request you to mould the design a bit so that we build something on the top of what we already have. Btw this design is good enough to take into consideration so feel free to come up with a PR :)

**Comment by yashsehgal** [comment]
> Hey @yashsehgal this design is nice and clean! But I request you to mould the design a bit so that we build something on top of what we already have. Btw this design is good enough to take into consideration so feel free to come up with a PR :)

Thanks. Shall I make a new PR or Should I first re-design it and then create one?

**Comment by Kajol-Kumari** [comment]
> > Hey @yashsehgal this design is nice and clean! But I request you to mould the design a bit so that we build something on top of what we already have. Btw this design is good enough to take into consideration so feel free to come up with a PR :)
> 
> Thanks. Shall I make a new PR or Should I first re-design it and then create one?

Try re-designing and then make the PR :)

**Comment by yashsehgal** [comment]
> > > Hey @yashsehgal this design is nice and clean! But I request you to mould the design a bit so that we build something on top of what we already have. Btw this design is good enough to take into consideration so feel free to come up with a PR :)
> > 
> > Thanks. Shall I make a new PR or Should I first re-design it and then create one?
> 
> Try re-designing and then make the PR :)

Okay

**Comment by yashsehgal** [comment]
This is the new one.

![homepage](https://user-images.githubusercontent.com/62352288/103747383-92fe8600-5028-11eb-95bd-c2246b1f34ec.png)
